### PR TITLE
Bump version to 3.0.0rc5

### PR DESCRIPTION
**Pull request by BIGWangYuDong**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,11 +94,11 @@ We are excited to announce our latest work on real-time object recognition tasks
 <img src="https://user-images.githubusercontent.com/12907710/208044554-1e8de6b5-48d8-44e4-a7b5-75076c7ebb71.png"/>
 </div>
 
-**v3.0.0rc4** was released in 25/11/2022:
+**v3.0.0rc5** was released in 26/12/2022:
 
-- Support [CondInst](https://arxiv.org/abs/2003.05664)
-- Add [`projects/`](./projects/) folder and give an [example](./projects/example_project/README.md) for communities to contribute their projects.
-- Support [SparseInst](https://arxiv.org/abs/2203.12827) in [`projects`](./projects/SparseInst/README.md)
+- Support [RTMDet-Ins](https://arxiv.org/abs/2212.07784) and improve RTMDet test config
```
Comment (ZwwWayne) [inline README.md:99]: also mention the technical report

File: README.md [hunk 2]
```diff
@@ -94,11 +94,11 @@ We are excited to announce our latest work on real-time object recognition tasks
...
+- Support [SSH](https://arxiv.org/abs/1708.03979)
```
Comment (ZwwWayne) [inline README.md:100]: some people may not understand what does SSH mean

File: README.md [hunk 3]
```diff
@@ -94,11 +94,11 @@ We are excited to announce our latest work on real-time object recognition tasks
...
+- Release [CondInst](https://arxiv.org/abs/2003.05664) pre-trained model
```
Comment (ZwwWayne) [inline README.md:101]: can omit this if we only release model

File: README.md (hunk 2 above)
Comment (RangiLyu) [inline README.md:100]: only support the module, not the whole algorithm.

**Review by RangiLyu — DISMISSED** [review]

File: README.md [hunk 4]
```diff
@@ -94,11 +94,11 @@ We are excited to announce our latest work on real-time object recognition tasks
...
 </div>
 
-**v3.0.0rc4** was released in 25/11/2022:
+**v3.0.0rc5** was released in 26/12/2022:
 
-- Support [CondInst](https://arxiv.org/abs/2003.05664)
-- Add [`projects/`](./projects/) folder and give an [example](./projects/example_project/README.md) for communities to contribute their projects.
-- Support [SparseInst](https://arxiv.org/abs/2203.12827) in [`projects`](./projects/SparseInst/README.md)
+- Release the technical report of [**RTMDet**](https://arxiv.org/abs/2212.07784)
+- Support [RTMDet](https://arxiv.org/abs/2212.07784) instance segmentation models
```
Comment (ZwwWayne) [inline README.md:100]:
```suggestion
- Support [RTMDet](https://arxiv.org/abs/2212.07784) instance segmentation models. The technical report of RTMDet is on [arxiv](https://arxiv.org/abs/2212.07784)
```

File: docs/en/notes/changelog.md [hunk 5]
```diff
@@ -1,5 +1,65 @@
...
+- Refactor `setup.cfg` (#9370)
+- Clip saturation value to valid range `[0, 1]` (#9391)
+- Only keep meta and state_dict when publish model (#9356)
+- Add segm evaluator in ms-poly_3x_coco_insatnce config (#9524)
+- Update deployment guide (#9527)
+- Update zh_cn `faq.md` (#9396)
+- Update `get_started` (#9480)
+- Update the zh_cn user_guides of `useful_tools.md` and `useful_hooks.md` (#9453)
+- Add type hints for `bfp` and `channel_mapper` (#9410)
+- Add type hints of part 8 including `iou_loss.py`, etc. (#9397)
```
Comment (ZwwWayne) [inline docs/en/notes/changelog.md:46]: what does `add type hint of part 8 means`?

Comment (ZwwWayne) [reply docs/en/notes/changelog.md:46]: Use and update the description of each commit from the commit messages

File: docs/en/notes/changelog.md [hunk 6]
```diff
@@ -1,5 +1,65 @@
...
+### Bug Fixes
+
+- Fix Condinst predict error when val `batch_size` > 1 (#9400)
+- Fix browse dataset (#9401)
+- Fix `analyze_logs.py` to plot mAP and calculate train time correctly (#9409)
+- Fix backward inplace error with `PAFPN` (#9450)
+- Fix config import links in model converters (#9441)
+- Fix `DeformableDETRHead` object has no attribute `loss_single` (#9477)
+- Fix the logic of pseudo bboxes predicted by teacher model in SemiBaseDetector (#9414)
+- Fix mmdet3.x InstanceSeg_Tutorial demo api (#9226)
```
Comment (ZwwWayne) [inline docs/en/notes/changelog.md:25]: Fix demo API in instance segmentation tutorial

File: docs/en/notes/changelog.md [hunk 7]
```diff
@@ -1,5 +1,65 @@
...
+
+### New Features
+
+- Support [RTMDet](https://arxiv.org/abs/2212.07784) instance segmentation models and improve RTMDet test config (#9494)
+- Support SSHContextModule in paper [SSH: Single Stage Headless Face Detector](https://arxiv.org/abs/1708.03979) (#8953)
+- Release [CondInst](https://arxiv.org/abs/2003.05664) pre-trained model (#9406)
+
+### Bug Fixes
+
+- Fix Condinst predict error when val `batch_size` > 1 (#9400)
```
Comment (ZwwWayne) [inline docs/en/notes/changelog.md:18]: when `batch_size` is greater than 1 in inference

File: docs/en/notes/changelog.md [hunk 8]
```diff
@@ -1,5 +1,65 @@
...
+- Fix `analyze_results` (#9380)
+- Fix the error that Readthedocs API cannot be displayed (#9510)
+
+### Improvements
+
+- Remove legacy `builder.py` (#9479)
+- Make sure the pipeline input shape is in WH (#9324)
+- Add `.pre-commit-config-zh-cn.yaml` file (#9388)
+- Refactor dataset metainfo to lowercase (#9469)
+- Add torch1.13 checking in CI (#9478)
```
Comment (ZwwWayne) [inline docs/en/notes/changelog.md:35]: Add PyTorch 1.13 checking in CI.

File: docs/en/notes/changelog.md [hunk 9]
```diff
@@ -1,5 +1,65 @@
 # Changelog of v3.x
 
+## v3.0.0rc4 (26/12/2022)
```
Comment (Li-Qingyun) [inline docs/en/notes/changelog.md:3]: rc5

File: docs/en/notes/changelog.md [hunk 10]
```diff
@@ -1,5 +1,65 @@
...
+
+### New Features
+
+- Support [RTMDet](https://arxiv.org/abs/2212.07784) instance segmentation models and improve RTMDet test config (#9494)
+- Support SSHContextModule in paper [SSH: Single Stage Headless Face Detector](https://arxiv.org/abs/1708.03979) (#8953)
+- Release [CondInst](https://arxiv.org/abs/2003.05664) pre-trained model (#9406)
+
+### Bug Fixes
+
+- Fix Condinst predict error when `batch_size` is greater than 1 in inference (#9400)
```
Comment (ZwwWayne) [inline docs/en/notes/changelog.md:18]: Condinst -> CondInst

File: docs/en/notes/changelog.md [hunk 11]
```diff
@@ -1,5 +1,65 @@
...
+- Fix browse dataset (#9401)
```
Comment (ZwwWayne) [inline docs/en/notes/changelog.md:19]: Fix what bug in browse_dataset.py?

File: docs/en/notes/changelog.md [hunk 12]
```diff
@@ -1,5 +1,65 @@
...
+- Fix `DeformableDETRHead` object has no attribute `loss_single` (#9477)
+- Fix the logic of pseudo bboxes predicted by teacher model in SemiBaseDetector (#9414)
+- Fix demo API in instance segmentation tutorial (#9226)
+- Fix `analyze_results` (#9380)
+- Fix the error that Readthedocs API cannot be displayed (#9510)
+
+### Improvements
+
+- Remove legacy `builder.py` (#9479)
+- Make sure the pipeline argument shape is in WH order (#9324)
```
Comment (ZwwWayne) [inline docs/en/notes/changelog.md:32]: in `(width, height)` order

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**